examples/with-claims/api/claims/EmailVerifiedClaim.ts

```diff
@@ -0,0 +1,30 @@
+import { SessionClaimValidator } from "supertokens-node/recipe/session";
+import { BooleanClaim } from "supertokens-node/lib/build/recipe/session/claimBaseClasses/booleanClaim";
+import { isEmailVerified } from "supertokens-node/recipe/emailpassword";
+
```

Comment:
If this is claim is not met, we want to automatically navigate the user to the mail verification screen - the user should not have to manually check sessionContext and navigate there.

This is unlike other claims (like Roles) where the user has to decide what to do in case a role is not met